nixos/modules/services/x11/display-managers/sddm.nix

```diff
@@ -69,6 +70,13 @@ let
       Session = autoLoginSessionName;
       Relogin = cfg.autoLogin.relogin;
     };
+  } // lib.optionalAttrs cfg.enableKwinWayland {
```

Comment:
It should probably merge subattrs instead as it currently replaces the content of General and Wayland groups completely